content/docs/guides/job-scheduling.md

```diff
@@ -0,0 +1,101 @@
++++
+title = "Job Scheduling"
+description = "Schedule job with gang-scheduling"
+weight = 10
+toc = true
+[menu.docs]
+  parent = "guides"
+  weight = 30
++++
+
```

Comment:
It'd be great to add a short introduction before the first heading, describing the content of this page. Something like "This page tells you how to add gang scheduling to a job, to allow jobs to run multiple pods at the same time."